src/discretization.jl

```diff
@@ -108,6 +108,9 @@ discretize(ball::Ball{2}) =
 discretize(sphere::Sphere{3}) =
   discretize(sphere, RegularDiscretization(50))
 
+discretize(torus::Torus) =
+  discretize(torus, RegularDiscretization(60, 30))
```

Comment:
```suggestion
  discretize(torus, RegularDiscretization(50))
```
We default to 50 samples per parametric dimension. Is it too bad for visualization?

Reply:
Didn't try but the second parameter is for the small circle (minor radius) and the first parameter for the big circle (major radius) so it seems sensible to take more subdivisions for the first.

Reply:
I don't know if we should add an exception. It seems a bit arbitrary anyways. Let's leave as the others and we can always fine tune later.